docs/_scripts/macros.py

```diff
@@ -45,19 +46,20 @@ def define_env(env: macros.MacrosPlugin) -> None:
     env.variables["code_annotation_marker"] = _CODE_ANNOTATION_MARKER
 
     @env.macro  # type: ignore[misc]
-    def glossary(term: str) -> str:
+    def glossary(term: str, text: str | None = None) -> str:
         """Create a link to the glossary entry for the given term.
 
         Args:
             term: The term to link to.
+            text: The text to display for the link. Defaults to the term.
 
         Returns:
             The Markdown link to the glossary entry for the given term.
         """
         current_path = pathlib.Path(env.page.file.src_uri)
         glossary_path = pathlib.Path("user-guide/glossary.md")
-        link_path = glossary_path.relative_to(current_path.parent)
-        return f"[{term}]({link_path}#{_slugify(term)})"
+        link_path = os.path.relpath(glossary_path, current_path.parent)
```

Comment:
Mmm, interesting. I would add a comment about this because it's not obvious at all so I wouldn't be surprised if someone wants to change it back.

Reply:
done